README.md

```diff
@@ -1,79 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
```

Comment:
I don't really like saying we provide visualizations similar to tensorflow yet cause we don't, and even if we did it would be the web / mobile U.I's doing it not the SDK, but maybe its fine since it gets the point across?

Reply:
We are conceptually moving towards Tensorboards visualizations, though we have a lot of distance to cover. I think it's reasonable to state the direction of work, as long as we're candid about where we are (via screenshots, examples, etc.) I'll play with the wording to make that more clear.